Travis + Octave + Demo folder + Model issues

tanguyduval opened the pull request.

tanguyduval commented:
@ileppe this branch is merged with your branch doc. Can you check if it's correct (and maybe modify it). I think we need to change the following:
  - We have 2 folders for doc (docs/Documentation), can we move doc to Documentation/sphinx?
  - Do you have a URL for the doc that we can add in the readme?

Can you please approve after review so that I can merge with master? Many thanks

ileppe commented:
@tanguyduval according to ReadtheDocs:
'we will first look for a conf.py file in your repository. If we don’t find one, we will generate one for you. We will look inside a doc or docs directory first, and then look within your entire project'
So it is easier to leave it in the 'doc' folder. Also, if I'm not mistaken, if we decide to go with github.io, the html must be in a folder called 'doc'. I would suggest putting everything in the 'doc' folder

ileppe commented:
@tanguyduval 
_._ for now the URL for the documentation is :http://qmrlab.readthedocs.io/en/doc/documentation.html
but this will change once it's merged into master (http://qmrlab.readthedocs.io/en/documentation.html)

tanguyduval commented:
ok I corrected Readme and moved doc in 1671eff1c5b16d6427a6a650bc9f782b7207c846 and 4bd3dadf4f8014f5cff8afe63ecb73ed69a1f3e7.
Can you approve the pull request (Travis is currently failing due to Octave servers... hopefully that's temporary)

ileppe commented:
@tanguyduval and fix conf.py pls

tanguyduval commented:
ok done in b52b9988d205a2479334e8a31e3c42fc30b2d5cd

tanguyduval commented:
Travis fails because sourceforge is down today:
https://twitter.com/sfnet_ops?ref_src=twsrc%5Etfw&ref_url=https%3A%2F%2Fsourceforge.net%2Ferror-404.html

tanguyduval commented on docs/source/conf.py:
```diff
@@ -0,0 +1,203 @@
+     'Miscellaneous'),
+]
+
+import xml.etree.ElementTree
```
@ileppe the last version of conf.py does not include these lines. We should be good for now
